lusSTR/wrappers/convert.py

```diff
@@ -108,9 +108,31 @@ def format_table(input, software, kit="forenseq"):
                         "Please check STRait Razor version!!"
                     )
                     print(msg)
+        if (
+            locus == "PENTA D"
+            and kit == "powerseq"
+            and marker.indel_flag == "Possible indel or partial sequence"
+        ):
+            marker = check_pentad(marker, sequence, software)
+            indel_flag = "Possible indel or partial sequence"
+        elif (
+            locus == "D7S820"
+            and kit == "powerseq"
+            and marker.indel_flag == "Possible indel or partial sequence"
+        ):
```

Comment:
Checking for the indel flag in each of these conditionals and then assigning it to an accessory variable in each block is redundant and unnecessary. You should check, but I think the following code should accomplish the same thing.

```diff
                         "Please check STRait Razor version!!"
                     )
                     print(msg)
-        if (
-            locus == "PENTA D"
-            and kit == "powerseq"
-            and marker.indel_flag == "Possible indel or partial sequence"
-        ):
-            marker = check_pentad(marker, sequence, software)
-            indel_flag = "Possible indel or partial sequence"
-        elif (
-            locus == "D7S820"
-            and kit == "powerseq"
-            and marker.indel_flag == "Possible indel or partial sequence"
-        ):
-            marker = check_D7(marker, sequence, software)
-            indel_flag = "Possible indel or partial sequence"
-        elif (
-            locus == "VWA"
-            and kit == "powerseq"
-            and marker.indel_flag == "Possible indel or partial sequence"
-        ):
-            marker = check_vwa(marker, sequence, software)
-            indel_flag = "Possible indel or partial sequence"
-        else:
-            indel_flag = marker.indel_flag
+        indel_flag = marker.indel_flag
+        if indel_flag == "Possible indel or partial sequence":
+            if locus == "PENTA D" and kit == "powerseq":
+                marker = check_pentad(marker, sequence, software)
+            elif locus == "D7S820" and kit == "powerseq":
+                marker = check_D7(marker, sequence, software)
+            elif locus == "VWA" and kit == "powerseq":
+                marker = check_vwa(marker, sequence, software)
         summary = [sampleid, project, analysis, locus] + marker.summary + [reads]
         list_of_lists.append(summary)
         if software != "uas":
```